core/tests/unit/ldml/test_unicode.cpp

```diff
@@ -82,51 +87,62 @@ std::string get_block_unicode_ver(const char *blocks_path) {
   assert(std::getline(blocks_file, block_line));  // first line
   const std::string prefix = "# Blocks-";
   assert(block_line.length() > prefix.length());
-  return block_line.substr(prefix.length());
+  std::string result = block_line.substr(prefix.length());
+  const std::string txt_suffix = ".txt"; // trim off this suffix
+  auto txt_pos = result.find(txt_suffix, 0);
+  if (txt_pos != std::string::npos) {
+    result.resize(txt_pos);
+  }
+  return result;
```

Comment:
I don't understand this change?

Reply:
Maybe needs a comment! It drops the ".txt" suffix that is present in the original file.

Reply:
Previous output without this change:

> block_unicode_ver	15.1.0.txt

Reply:
Gotcha, a comment would be nice to clarify why this is being done.